content/community/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Node.js Community
+description: "Node.js has one of the largest, most vibrant and innovative open source communities in the world. Node.js is built, used, and maintained by the community. We believe the community to be at the core of Node.js popularity. Amazing people from every part of the world, with a common interest, collaborate and shape this community."
+authors: ahmadawais, maedahbatool, saqibameen, msaaddev
+---
+
+## Introduction
+
+Node.js has one of the largest, most vibrant and innovative open source communities in the world. Node.js is built, used, and maintained by the community. We believe the community to be at the core of Node.js popularity. Amazing people from every part of the world, with a common interest, collaborate and shape this community.
+
+## Node.js Contributor Covenant Code of Conduct
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+[Read the full code of conduct →](https://github.com/nodejs/admin/blob/master/CODE_OF_CONDUCT.md)
+
+## Ways to Contribute
+
+We welcome all kinds of contributions — documentation, bugs reporting, core contribution. Irrespective of the extent of the contribution, we encourage you to be a part of Node.js. If you miss a feature, request it, or maybe help build it.
+
+* [Community Discussions](https://nodejs.org/en/get-involved/#community-discussion)
+* [Learning Node.js](https://nodejs.org/en/get-involved/#learning)
+* [Asking for General Help](https://nodejs.org/en/get-involved/contribute/#asking-for-general-help)
+* [Reporting an Issue](https://nodejs.org/en/get-involved/contribute/#reporting-an-issue)
+* [Code contributions](https://nodejs.org/en/get-involved/contribute/#code-contributions)
+* [Becoming a collaborator](https://nodejs.org/en/get-involved/contribute/#becoming-a-collaborator)
+* [Community Committee](https://github.com/nodejs/community-committee)
+* [Node.js Meetups](https://nodejs.org/en/get-involved/node-meetups/)
+* [Start Contributing to Node.js](https://dev.to/azure/start-contributing-to-nodejs-in-the-new-year-3dlh)
+
+## Node.js News
+
+### Stay in Touch
+
+Subscribe to the [Node.js monthly mailing list](https://us14.campaign-archive.com/home/?u=c7c2e114a827812354112c23b&id=f006b61f29) to get all the latest updates related to the Node.js.
+
+### Twitter
+
+For social updates, [follow @Nodejs on Twitter](https://twitter.com/nodejs).
+
+## Where to Get Support
```

Comment:
It might be a good idea to mention #Node.js on freenode IRC channel here, or maybe have a section above for "Real-time chat spaces", including IRC, Node Slackers and OpenJSF Slack.

Reply:
I agree. I'll create another PR for the content changes. Right now this new page will be in the codebase but wont be published until the design is completed.

Reply:
See https://github.com/nodejs/nodejs.dev/pull/1107